Makefile

```
@@ -193,6 +193,7 @@ csv-clean: ## Remove existing OLM files.
 crds: $(CONTROLLER_GEN) $(YQ)
 	@echo Updating CRD manifests
 	@build/crds/build-crds.sh $(CONTROLLER_GEN) $(YQ)
+	@build/crds/generate-crd-docs.sh
```

Try adding an env var here, which then you can implement in the script to override the default behavior of only generating it if types.go has changed
```suggestion
	@export FORCE_GEN_CRD_DOCS=true 
	@build/crds/generate-crd-docs.sh
```

---

Thanks, added the changes, they worked